### PR TITLE
fix Field 'name' in type 'AppServerInfo' is undefined error in javascript

### PR DESCRIPTION
**Pull request by candyleer**
in web ui menu "Server",it throw an exception that is
```
Validation error of type FieldUndefined: Field 'name' in type 'AppServerInfo' is undefined
```
so i trace it in client js and server,i found that in this url
`http://localhost:8080/api/server/search`  as same as `http://localhost:8080/api/application` it's response is
```
{
    "errors": [
        {
            "message": "Validation error of type FieldUndefined: Field 'name' in type 'AppServerInfo' is undefined"
        }
    ]
}
```
continue, this is producted in Graphql in server-side,the `server-layer.graphql` definition of `AppServerInfo` is
```
type AppServerInfo {
    id: ID!
    osName: String!
    applicationId: Int!
    applicationCode: String
    callsPerSec: Int!
    host: String
    pid: Int
    ipv4: [String!]!
}
```
but in javascript.the field is `name` ,see `Server.js`,that is
```
 query={`
                  query SearchServer($keyword: String!, $duration: Duration!) {
                    searchServer(keyword: $keyword, duration: $duration) {
                      key: id
                      name
                      host
                      pid
                      ipv4
                      applicationCode
                    }
                  }
                `}
```

the field is `name`,not 'osName' in server's graphql definition,so this error occur.
In summary, the data structure of the server and client are inconsistent

### CODE REVIEW

**Comment by wu-sheng** [comment]
@candyleer I think you are faster than our PMC :) This is a new version(beta) feature, check here: https://github.com/apache/incubator-skywalking/issues/998

Of course, glad to have this pr.

This pr need some updates. We should change os in page to this field. After that, please post a screenshot in the comments.

**Comment by wu-sheng** [comment]
I mean here:

image

**Comment by candyleer** [comment]
@wu-sheng  haha ,i have updated this `serverInfo.osName` just now , and now it's worked
image
